src/components/recruit/recruitBoard/detailedFilter/DetailedInput.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useState, useEffect } from 'react';
+import { Search, XBtn, Clear } from '../../../../assets';
+import { useQuery } from '@tanstack/react-query';
+import { getRoleKeyword, getSkillKeyword, getTagKeyword } from '../../../../service';
+import { useDebounce } from '../../../../hooks';
+import { Keyword, DetailedInfo, Array } from '../../../../types';
+import { useSetRecoilState } from 'recoil';
+import { recruitFilterState } from '../../../../atom';
+
+const SKILL_MESSAGE_OBJ = {
+	intro: '보유하신 기술 스택을 검색해보세요. 기술 스택은 최대 5개까지 선택 가능합니다.',
+	message: '기술 스택을 검색하세요.',
+};
+
+const ROLE_MESSAGE_OBJ = {
+	intro: '원하는 역할을 검색해보세요. 역할은 최대 5개까지 선택 가능합니다.',
+	message: '역할을 검색하세요.',
+};
+
+const TAG_MESSAGE_OBJ = {
+	intro: '알아보고 싶은 내용을 태그로 검색해보세요. 태그는 최대 5개까지 선택 가능합니다.',
+	message: '태그를 검색하세요.',
+};
```

Comment:
```suggestion
const MESSAGE = {
    skill: {
        intro: '보유하신 기술 스택을 검색해보세요. 기술 스택은 최대 5개까지 선택 가능합니다.',
	    message: '기술 스택을 검색하세요.',
    }, 
    role: {
        intro: '원하는 역할을 검색해보세요. 역할은 최대 5개까지 선택 가능합니다.',
	    message: '역할을 검색하세요.',
    },
    tag: {
        intro: '알아보고 싶은 내용을 태그로 검색해보세요. 태그는 최대 5개까지 선택 가능합니다.',
	    message: '태그를 검색하세요.',
    }
}
```

한번에 상수화 시키면 좋을 것 같은데 어떠신가용??

Reply:
넵 좋은 생각인 것 같아요!!
상수 선언이 적으면 좋으니까요!!